### ISSUE TITLE
poc: draft / kubernetes integration

### ISSUE THREAD

**Issue by rjeczalik** [koding/koding#11303]
Purpose of this feature is to discover possible interop between draft / kd / koding. In order to have a shot at it, we may want first to make it possible to install draftd / helm on k8s cluster built by koding.

Prerequisite list to get there:

- [ ] kloud/stack: add API for deploying klient on an existing vm (for connecting k8s cluster nodes)
- [ ] kloud/provider: augmentation for cluster resources 
  - [ ] provider/google: augment google_container_cluster resource
  - [ ] provider/aws: augment for aws_ecs_cluster resource
  - [ ] provider/azurerm: augment azurerm_container_service service
    - [ ] provider: add azurerm provider
- [ ] klient: add proxy for connecting to k8s over kite
  - [ ] kloud: add k8s proxy

**Comment by tardyp** [comment]
@rjeczalik did you have time to work on this? any code to share?

I was planning to assign @protatremy from my team to work on it. We probably were not as ambitious as this plan though.
We were planning to start from the marathon backend, and see how we can make it work on k8s.
Then gradually improve on making it right.

**Comment by rjeczalik** [comment]
@tardyp No, unfortunately, I've put it on hold and focused on other issue.

What's you goal with k8s integration? I could explain what I was trying to achieve here if you' like.

**Comment by xeor** [comment]
*a friendly bump*
